src/azure-cli-core/azure/cli/core/tests/test_profile.py

```diff
@@ -741,89 +741,6 @@ def test_get_login_credentials_for_graph_client(self, mock_get_token, mock_read_
-        test_sub = Subscription()
-        setattr(test_sub, 'id', 'id123')
-        setattr(test_sub, 'subscription_id', 'id123')
-        setattr(test_sub, 'display_name', 'good name')
-        setattr(test_sub, 'state', SubscriptionState.enabled)
-        setattr(test_sub, 'tenant_id', '54826b22-38d6-4fb2-bad9-b7b93a3e9c5a')
-
-        with mock.patch('azure.cli.core._profile.SubscriptionFinder._find_using_specific_tenant', autospec=True, return_value=[test_sub]):
-            profile = Profile(cli_ctx=cli, async_persist=False, storage=test_account)
-            result_accounts = profile.find_subscriptions_in_cloud_console([arm_token, kv_token])
-
-        # verify the local account
-        expected_subscription = {
-            "state": "Enabled",
-            "user": {
-                "type": "user",
-                "name": "admin3@AzureSDKTeam.onmicrosoft.com"
-            },
-            "name": "good name",
-            "isDefault": True,
-            "id": "id123",
-            "environmentName": "AzureCloud",
-            "tenantId": "54826b22-38d6-4fb2-bad9-b7b93a3e9c5a"
-        }
-
-        actual = get_file_json(test_token_file)
-
-        self.assertEqual([expected_subscription], result_accounts)
-        # sanity check that the expiration time is about 45 minutes away
-        self.assertTrue(parser.parse(actual[0]['expiresOn']) < datetime.now() + timedelta(minutes=45))
-        self.assertTrue('oid' not in actual[0])
```

Comment:
why is this test being removed?

Reply:
The cloud shell login flow this test targets is gone as we are moving to msi model, hence i just remove the test